android-components/components/service/contile/src/main/java/mozilla/components/service/contile/ContileTopSitesProvider.kt

```diff
@@ -44,14 +45,17 @@ internal const val CACHE_TOP_SITES_KEY = "tiles"
  */
 class ContileTopSitesProvider(
     context: Context,
+    val isMozillaOnline: Boolean = false,
```

Comment:
❓ should we be passing this value around?  Since we can get this through `Config.channel.isMozillaOnline`?

Reply:
I'm not sure if we can get `Config.channel.isMozillaOnline` in **android-components** ? I've only used that in **fenix** itself.

Reply:
I see, that makes sense.  Thanks